### PR TITLE
Use empty key mapping if US layout is detected

### LINKED ISSUE #4986: [keybinding] Support international/US layout
see https://github.com/theia-ide/theia/pull/4998#discussion_r278448654

### PR DESCRIPTION
**Pull request by spoenemann**
Fixes #4986.

### CODE REVIEW

**Comment by AlexTugarev** [comment]
Testing right now...

File: packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts [hunk 1]
```diff
@@ -73,11 +73,12 @@ export class BrowserKeyboardLayoutProvider implements KeyboardLayoutProvider, Ke
         for (const [code, key] of layoutMap.entries()) {
             tester.updateScores({ code, key });
         }
-        const result = tester.getTopScoringCandidates();
-        if (result.length > 0) {
-            return result[0];
+        const topScoring = tester.getTopScoringCandidates();
+        const result = topScoring.find(l => l !== this.macUS && l !== this.winUS);
+        if (result) {
```
Comment (AlexTugarev) [inline packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts]: `result` is `undefined`

Screen Shot 2019-04-25 at 09 57 45

Comment (spoenemann) [reply packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts]: Yes, that's intended. In that case it should fall back to `this.createDefaultLayout()`.

Comment (AlexTugarev) [reply packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts]: @spoenemann, can you check the predicate of `find` it looks bogus.

Comment (spoenemann) [reply packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts]: The problem is, there are different US layouts :-D

You seem to have the `` ` `` key between `shift` and `Z`, but on the static layout I created it's left of `1` (where you have `§`).

Comment (spoenemann) [reply packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts]: With my change an empty mapping is used, so no key is remapped.

If that does still not work, you need to create a static layout with your keyboard.

Comment (AlexTugarev) [reply packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts]: @spoenemann, you're right! I'll create such a static layout.

this is from the MBP keeb:
Screen Shot 2019-04-25 at 10 34 38

this is from my external keeb:
Screen Shot 2019-04-25 at 10 33 30

**Comment by spoenemann** [comment]
It seems that Chrome reports the `` ` `` key as `IntlBackslash`. Creating a new static layout won't help. I'll give it another try.

**Pull request closed without merging**